trace/trace_opentelemetry/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START trace_opentelemetry]
+
+func main() {
```

Comment:
Assuming this is a quickstart, you'll want something like the following to fix the compilation issue.

`trace_quickstart` is the name of the binary, which defaults to the name of the directory this is in. So, that will need to be updated based on the directory naming. See https://github.com/golang/go/wiki/CodeReviewComments#package-comments.

```suggestion
// The trace_opentelemetry command is an example of setting up OpenTelemetry to export traces to Stackdriver.
package main

func main() {
```